base/strings/io.jl

```diff
@@ -382,12 +383,14 @@ julia> unescape_string("\\\\101") # octal
 ## See also
 [`escape_string`](@ref).
 """
-function unescape_string(io, s::AbstractString)
+function unescape_string(io, s::AbstractString, keep = ())
```

Comment:
Add `io::IO` to fix the ambiguity error which caused CI failure (and adjust docstring).